dotCMS/src/main/java/com/dotmarketing/db/DotDatasourceStrategy.java

```diff
@@ -0,0 +1,8 @@
+package com.dotmarketing.db;
+
+import javax.sql.DataSource;
+
+public interface DotDatasourceStrategy {
```

Comment:
this could be FunctionalInterface since it is just want method, also add some doc